FLAnimatedImage/IndexedImageCache.m

```diff
@@ -0,0 +1,107 @@
+//
+//  BlendedImageCache.m
+//  FLAnimatedImage
+//
+//  Created by Sih Ou-Yang on 2024/2/23.
+//  Copyright © 2024 com.cardinalblue. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+@interface IndexedImageInfo : NSObject
```

Comment:
[Suggestion] How about writing this in swift :P I guess we would love this code base to evolve from Objc.

Reply:
Yeah, it would be more simpler in swift as well. I think I'll do it in another branch 👍